x-pack/platform/test/onechat_api_integration/apis/esql_tools.ts

```diff
@@ -125,6 +125,43 @@ export default function ({ getService }: FtrProviderContext) {
       });
     });
 
+    describe('POST /api/chat/tools/_execute', () => {
+      before(async () => {
+        const testTool = {
+          type: 'esql',
+          description: 'A test tool',
+          tags: ['test'],
+          configuration: {
+            query: 'FROM .internal.alerts-observability.logs.alerts-default-000001 | LIMIT 3',
```

Comment:
This is one of the indices available in the test environment, however I think it means we depend on some other team to keep it there? I couldn't find how we could add our own and with what data.

Reply:
This is risky, what if they change the index pattern/name etc. 

IMO better to create our own index with dummy data and then clean this up after the test? It should be possible since FTRs are running against real ES instance afaik